### PR TITLE
Add a credit screen and perform bug fixes

### PR DESCRIPTION
**Pull request by Dennisbonke**
This PR adds a credit screen, made it accessible from the main menu, adds a continue button on the main menu and fixes a bug in level loading.

### CODE REVIEW

**Review by drfdaniel — DISMISSED** [review]
LGTM

File: Assets/Scripts/Core/Controllers/CreditController.cs [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+		private void Update() {
+			HandleMisc();
+		}
+		private void HandleMisc() {
+			if(Input.GetKeyDown(KeyCode.Escape)) {
+				GlobalController.instance.ReturnMainMenu();
+			}
+		}
+	}
+}
```
Comment (Westlanderz) [inline Assets/Scripts/Core/Controllers/CreditController.cs:14]: No new line at the end of the file.

File: Assets/Scripts/Core/Controllers/CreditController.cs [hunk 2]
```diff
@@ -0,0 +1,14 @@
+using UnityEngine;
+
+namespace OperationBlackwell.Core {
+	public class CreditController : Singleton<CreditController> {
+		private void Update() {
+			HandleMisc();
+		}
```
Comment (Westlanderz) [inline Assets/Scripts/Core/Controllers/CreditController.cs:7]: New lines are supposed to be between functions.

**Review by Westlanderz — CHANGES_REQUESTED** [review]
LGTM apart from a couple of nitpicks

**Review by Westlanderz — APPROVED** [review]
lgtm

**Pull request merged**